hackathon_site/registration/models.py

```diff
@@ -25,23 +25,65 @@ def __str__(self):
 
 
 class Application(models.Model):
+    GENDER_CHOICES = [
+        (None, ""),
+        ("male", "Male"),
+        ("female", "Female"),
+        ("non-binary", "Non-binary"),
+        ("other", "Other"),
```

Comment:
I figured no, since it's not a statistic that we've used in the past and it was extra work to implement. If users of the template want, they can remove the choices and leave it as just a text field, and use something like select2 with [tagging](https://select2.org/tagging) to have options available to start (like we do with the school field).

Reply:
`If users of the template want, they can remove the choices and leave it as just a text field, and use something like select2 with tagging to have options available to start (like we do with the school field).`

Maybe we should add this to the README

Reply:
Would be good to make a docs section with all of this: https://github.com/ieeeuoft/hackathon-template/issues/150